content/en/docs/reference/access-authn-authz/validating-admission-policy.md

```diff
@@ -438,7 +438,20 @@ For example, here is an admission policy with an audit annotation:
 
 When an API request is validated with this admission policy, the resulting audit event will look like:
 
-{{< codenew file="access/audit-event-with-audit-annotation.yaml" >}}
+```
```

Comment:
This is another "bad" example that is not intended for users to try out.

Reply:
Could mark this as YAML, if we like.

Reply:
It is not a valid YAML, just a snippet.